.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,36 @@
+name: Benchmark
+
+on:
+  push:
+    branchs: [ master ]
+  pull_request:
+
+jobs:
+  benchmark:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: ['10.x', '12.x', '14.x']
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache NPM dependencies
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-npm-cache
+          restore-keys: |
+            ${{ runner.OS }}-npm-cache
+      - name: Running benchmark
+        run: |
+          echo "============== Hexo Benchmark ==============" > benchmark-result-${{ matrix.node-version }}.txt
+          node test/benchmark.js >> benchmark-result-${{ runner.OS }}-${{ matrix.node-version }}.txt
+      - uses: actions/upload-artifact@v2
+        with:
+          name: benchmark-results
+          path: benchmark-result-${{ runner.OS }}-${{ matrix.node-version }}.txt
```

Comment:
IMHO, we shouldn't redirect the result into a text file and upload to `artifact`. Personally I hate download a file just to see the benchmark result.

Reply:
This benchmark result is the same as the current one?

```
Cold processing
...
Hot processing
...
Another Cold processing
...
```

If that's the case, I agree it's better to just stdout.

Reply:
@curbengh Yes. still `console.table`.